### PR TITLE
Add offset to frommandel

### PR DESCRIPTION
**Pull request by KeitaNakamura**
I added `offset` keyword argument to `frommandel` function.

### CODE REVIEW

**Comment by KeitaNakamura** [comment]
Tests failed at `test_misc.jl:313` which is for `eigfact`. Did I do something wrong?

**Comment by KristofferC** [comment]
No, just looks like the numeric tolerances on 32 bit is a bit strict?

File: src/voigt.jl [hunk 1]
```diff
@@ -131,6 +131,10 @@ Base.@propagate_inbounds @inline function fromvoigt(TT::Type{<: SymmetricTensor{
         end)
 end
 
-Base.@propagate_inbounds @inline function frommandel(TT::Union{Type{<: SymmetricTensor{2}}, Type{<: SymmetricTensor{4}}}, v::AbstractVecOrMat{T}) where {T}
-    fromvoigt(TT, v, offdiagscale = T(√2))
+Base.@propagate_inbounds @inline function frommandel(TT::Type{<: SymmetricTensor{2, dim}}, v::AbstractVector{T}; offset::Int = 1) where {dim, T}
```
Comment (fredrikekre) [inline src/voigt.jl]: there is no need for the `dim` parameter here

File: src/voigt.jl [hunk 2]
```diff
@@ -131,6 +131,10 @@ Base.@propagate_inbounds @inline function fromvoigt(TT::Type{<: SymmetricTensor{
...
+    fromvoigt(TT, v, offdiagscale = T(√2), offset = offset)
+end
+
+Base.@propagate_inbounds @inline function frommandel(TT::Type{<: SymmetricTensor{4, dim}}, v::AbstractMatrix{T}; offset_i::Int = 1, offset_j::Int = 1) where {dim, T}
+    fromvoigt(TT, v, offdiagscale = T(√2), offset_i = offset_i, offset_j = offset_j)
```
Comment (fredrikekre) [inline src/voigt.jl]: same here

**Comment by KeitaNakamura** [comment]
Okey. Updated.

**Pull request merged**